packages/nextjs/app/example-ui/page.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Metadata, NextPage } from "next";
+import { ContractData } from "~~/components/example-ui/ContractData";
+import { ContractInteraction } from "~~/components/example-ui/ContractInteraction";
+
+export const metadata: Metadata = {
+  title: "Example UI",
```

Comment:
btw why did you change title from `Example UI | Scaffold-ETH 2`?

Reply:
` title: {
    default: "Scaffold-ETH 2 App",
    template: "%s | Scaffold-ETH 2",
  },`
  I have used the above template in layout.tsx , so it basically is shown as `Example UI | Scaffold-ETH 2 `
 reference- https://nextjs.org/docs/app/api-reference/functions/generate-metadata#template